### DISCUSSION TITLE
delta Acceleration

### DISCUSSION THREAD

**Discussion by tggachilles** [AntiMicroX/antimicrox#787] [Q&A]
can anyone tell me exactly how the delta acceleration is coded, or where i can find the code. I would like to use it in a fps shooter im making in unity, but i cant get it to feel right

**Comment by pktiuk** [comment]
I guess you have to find it by yourself.  
When you will find that piece of code feel free to add some doxygen comments. Then it will appear in online docs: https://antimicrox.github.io/doxygen/
